src/main/java/frc/robot/commands/AutoBalance.java

```diff
@@ -47,39 +47,48 @@ public void initialize() {
   @Override
   public void execute() {
```

Comment:
Do we want to keep a constant heading so we don't get bumped and shoot off the platform?

Reply:
Yes, we can do that